### PR TITLE
REACTOR-408 - Clean up old reactor resource code path

### PR DESCRIPTION
**Pull request by tiyberius**

### CODE REVIEW

File: reactor/features/ingest.py [hunk 1]
```diff
@@ -46,36 +44,6 @@ def _write_events_to_bucket(bucket_name, events, message_source_environment_id,
...
-                cznef_events.append(normalized_event)
-
-    if cznef_events:
-        s3_provider = Provider(service='s3')
-        success = persist_fn(cznef_events, environment_id, namespace, reactor_account_id, s3_provider)
-        if not success:
-            logger.error('Failed to write flow log events to event bucket')
-
-    logger.info(f"Processed {len(cznef_events)} VPC FlowLog Events from {record.get('logGroup')}")
-    return cznef_events
```
Comment (killthrush) [inline reactor/features/ingest.py]: This needs to be put back.  It doesn't work anyway, so you can remove the reference to the old table and just exit early

File: reactor/features/resources/curate.py [hunk 2]
```diff
@@ -1,277 +0,0 @@
...
-        new_disposition = resource.DISPOSITION_ERROR
-        dbh.set_disposition(res_id, new_disposition, timestamp)
-        return "error"
-
-
-def generalized_enrichment(namespace, payload):
-    service_type = "general"
-    function_name = _ENRICH_FUNCTION_TEMPLATE.format(service_type)
-    lambda_utils.reactor_invoke(namespace, function_name, payload=json.dumps(payload), async=True)
-    return service_type
```
Comment (killthrush) [inline reactor/features/resources/curate.py]: yaaasssss

File: serverless.yml [hunk 3]
```diff
@@ -435,61 +420,11 @@ resources:
...
-            AttributeType: S
-        KeySchema:
-          - AttributeName: res_id
-            KeyType: HASH
-          - AttributeName: timestamp
-            KeyType: RANGE
-        ProvisionedThroughput:
-          ReadCapacityUnits: ${
-              self:custom.DB.Resources.read.minimum.${self:provider.stage},
-              self:custom.DB.Resources.read.minimum.default
```
Comment (killthrush) [inline serverless.yml]: you can also remove the parameters for this table (up near line 100 or so)

File: test/unit/features/test_ingest_unit.py [hunk 4]
```diff
@@ -4,11 +4,14 @@
...
 import base64
 import gzip
+
+import pytest
 import simplejson as json
 
 import reactor.features.ingest as ingest
 
 
+@pytest.mark.skip(reason='vpc_flow_log_events() doesn\'t really work')
```
Comment (killthrush) [inline test/unit/features/test_ingest_unit.py]: seems legit

File: lambda_handlers.py [hunk 5]
```diff
@@ -152,12 +151,6 @@ def ingest_events(event, context, namespace, configuration, client_details):
     return ingest.ingest_events(event.get('Records'), namespace, client_details)
 
 
-@lambda_handler()
-def ingest_vpc_flow_log_events(event, context, namespace, configuration, client_details):
-    reactor_account_id = get_account_id(context)
-    return ingest.ingest_vpc_flow_log_events(event, namespace, configuration, reactor_account_id)
```
[the remark this answers is not in the document]
Comment (tiyberius) [reply lambda_handlers.py]: Roger that, I'll put it back!

File: reactor/features/resources/curate.py (hunk 2 above)
Comment (tiyberius) [reply reactor/features/resources/curate.py]: see ya!!

**Pull request merged**